apps/emqx/src/emqx_ws_connection.erl

```diff
@@ -376,20 +377,21 @@ get_peer_info(Type, Listener, Req, Opts) ->
         emqx_config:get_listener_conf(Type, Listener, [proxy_protocol]) andalso
             maps:get(proxy_header, Req)
     of
-        #{src_address := SrcAddr, src_port := SrcPort, ssl := SSL} ->
+        #{src_address := SrcAddr, src_port := SrcPort, ssl := SSL} = ProxyInfo ->
             SourceName = {SrcAddr, SrcPort},
             %% Notice: Only CN is available in Proxy Protocol V2 additional info
```

Comment:
this comment is apparently wrong? it also returns authority.

Reply:
Does it mean that you can't get the DN through Proxy-Protocol?